src/test/java/org/apache/pulsar/io/jcloud/format/FormatTestBase.java

```diff
@@ -190,7 +190,7 @@ protected void validMetadata(org.apache.avro.generic.GenericRecord record, Messa
         );
         Assert.assertTrue(
                 mapsAreEqual(
-                        message.getReaderSchema().get().getSchemaInfo().getProperties(),
```

Comment:
@freeznet Maybe we could set the schema.properties and message.properties differently. test and verify both properties.

Reply:
@jiazhai thanks for your review, I have added some code blocks to set schemainfo properties and test with it, PTAL, thanks.